<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>340</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Output format:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_OutputFormat">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>99</y>
     <width>210</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_ChangeSamplerate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>170</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Resample the output file instead of keeping the samplerate of the input file.</string>
   </property>
   <property name="text">
    <string>Change samplerate</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Multithreading">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>350</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use all available CPU cores to convert multiple files at the same time.</string>
   </property>
   <property name="text">
    <string>Multithreading</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Output directory:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_OutputDirectory">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>59</y>
     <width>210</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>FFmpeg binary:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_FFmpegBinary">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>19</y>
     <width>210</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_SkipExistingFiles">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>350</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Do not convert already existing converted files again.
Disable this to reencode and overwrite existing files.</string>
   </property>
   <property name="text">
    <string>Skip existing files</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_OutputSamplerate">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>179</y>
     <width>170</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quality:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Quality">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>139</y>
     <width>210</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
